frontend/packages/core/src/AppLayout/user.tsx

```diff
@@ -48,7 +48,7 @@ const Paper = styled(MuiPaper)({
 
 const Popper = styled(MuiPopper)({
   padding: "0 12px",
-  marginLeft: "12px",
+  offset: "12px",
```

Comment:
for my understanding, why offset vs marginLeft?

Reply:
https://mui.com/material-ui/migration/v5-component-changes/#popper
https://popper.js.org/docs/v2/migration-guide/#4-remove-all-css-margins

The popper api was upgraded to v2 and margin was removed, has to be replaced with offset